src/components/my-map/index.ts

```diff
@@ -631,7 +648,11 @@ export class MyMap extends LitElement {
   render() {
     return html`<script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
       <link rel="stylesheet" href="https://cdn.skypack.dev/ol@^6.6.1/ol.css" />
-      <div id="${this.id}" class="map" tabindex="0" />`;
+      <div
+        id="${this.id}"
+        class="map"
+        tabindex="${this.staticMode ? -1 : 1}"
```

Comment:
```suggestion
        tabindex="${this.staticMode ? -1 : 0}"
```

See previous comment referencing MDN docs

Reply:
Also now fixed as result of above and using `-1 : 0` :white_check_mark: